controllers/operands/ssp.go

```diff
@@ -241,6 +241,8 @@ func getDataImportCronTemplates(hc *hcov1beta1.HyperConverged) ([]hcov1beta1.Dat
 	}
 	dictList = getCustomDicts(dictList, crDicts)
 
+	dictList = filterEnabledDicts(dictList)
+
 	sort.Sort(dataImportTemplateSlice(dictList))
```

Comment:
I would keep the filtering within `getCommonDicts` and add it also to `getCustomDicts`, avoiding building the list and then filter it, but build a pre-filtered list instead. WDYT? IMHO, it will save some memory and improve performance.

Reply:
Done